tests/setup/challenges-data.test.js

```diff
@@ -15,8 +14,8 @@ var challenge = [{
 var query = challengesData(challenge);
 
 var expected = 'INSERT INTO challenges '
```

Comment:
It would be good to add a test to check the value of the date when a challenge is added